testing/endpoint.go

```diff
@@ -569,7 +569,7 @@ func (endpoint *Endpoint) ChanUpgradeInit(timeout channeltypes.Timeout) error {
 		endpoint.ChannelConfig.PortID,
 		endpoint.ChannelID,
 		channeltypes.NewUpgradeFields(endpoint.ChannelConfig.Order, []string{endpoint.ConnectionID}, endpoint.ChannelConfig.Version),
-		channeltypes.NewUpgradeTimeout(timeout.Height, timeout.Timestamp),
+		channeltypes.NewTimeout(timeout.Height, timeout.Timestamp),
```

Comment:
I think we're passing in timeout as a function arg here and recreating a new timeout. We can use the arg directly